config/routes.rb

```diff
@@ -1,8 +1,6 @@
-Basechurch::Engine.routes.draw do
+Rails.application.routes.draw do
   resources :api_keys, except: [:new, :edit]
-  devise_for :users,
-             class_name: 'Basechurch::User',
-             controllers: { sessions: 'sessions' }
+  devise_for :users, controllers: { sessions: 'sessions' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.